acstools/acsphotcte.py

```diff
@@ -37,11 +37,11 @@
 >>> import numpy as np
 >>> from acstools import acsphotcte
```

Comment:
Might as well define a variable?
```suggestion
>>> from acstools import acsphotcte
>>> n_sample = 1000
```